docs/dictionary-data.md

```diff
@@ -4,7 +4,400 @@
 
```

Comment:
This file uses the `sssttt` convention a few times. The explainer in linguistic-data-intro.md about source/target language is good—reuse / link to that explanation in this file too?